client/src/modules/list/components/ListHeader.jsx

```diff
@@ -439,6 +447,10 @@ class ListHeader extends PureComponent {
       pendingForTypeUpdate
     } = this.state;
 
+    if (!locks) {
```

Comment:
Why are the locks blocking the rendering?

Reply:
When the app goes under /sack/id url it tries to render the List which contains ListHeader - it has no list details in first render, cause we will be fetching details on componentDidMount()
So without this locks data, app crashes, because it tries to destructure` {locks: name} ` where the name is undefined.